js/modules/k6/k6.go

```diff
@@ -104,7 +100,8 @@ func (mi *K6) Group(name string, val sobek.Value) (sobek.Value, error) {
 		return nil, errors.New("group() requires a callback as a second argument")
 	}
 	if common.IsAsyncFunction(mi.vu.Runtime(), val) {
-		return sobek.Undefined(), fmt.Errorf(asyncFunctionNotSupportedMsg, "group")
+		return sobek.Undefined(), errors.New("group() does not support async functions as arguments, " +
+			"please see https://grafana.com/docs/k6/latest/javascript-api/k6/group/#working-with-async-functions for more info")
```

Comment:
Just as a small nit, we don't have the heading #working-with-async-functions on that page, so you could just link to: https://grafana.com/docs/k6/latest/javascript-api/k6/group/.

Reply:
Good catch, thanks! @heitortsergent